app/src/main/java/com/sun/livescore/data/remote/ScoreDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.sun.livescore.data.remote
+
+import com.sun.livescore.data.model.score.fixture.DataFixture
+import com.sun.livescore.data.model.score.fixture.Fixture
+import com.sun.livescore.data.model.score.fixture.FixtureResponse
+import com.sun.livescore.data.model.score.history.DataHistory
+import com.sun.livescore.data.model.score.history.History
+import com.sun.livescore.data.model.score.history.HistoryResponse
+import io.reactivex.Single
+
+interface ScoreDataSource {
+    interface Remote {
+        fun getScoresFixtures(date: String): Single<FixtureResponse>
+        fun getScoresHistory(
```

Comment:
```suggestion
        fun getScoresHistory(from: String, to: String): Single<HistoryResponse>
```
cứ xuống dòng làm gì cho khó đọ